app/sidebar.py

```diff
@@ -8,8 +8,31 @@
 
 
 def make_sidebar(api: PtxboaAPI):
-    st.sidebar.subheader("Main settings:")
+    with st.sidebar.expander("**Main settings**", expanded=True):
+        main_settings(api)
 

```

Comment:
@markushal if you don't want the main settings in an expander, you can change to the following:
```suggestion
    with st.sidebar:
        st.subheader("Main settings")
        main_settings(api)

```